### DISCUSSION TITLE
SVG - Path Question

### DISCUSSION THREAD

**Discussion by gordon-brown** [clientIO/joint#2452] [Q&A]
### Introduction

I have some shapes I need to convert from our Syncfusion based application. These include clip are shapes as well as bpmn shapes. I have been able to convert the BPMN shapes, but the clip are is posing a challenage. The shape renders as a rectangle instead of the shape. I tried to do it similar to one of the Codepen samples. It is different in the pathdata doesn't include any calc functions. Not sure if this matters. The path data does seem to render correctly when I test it on svg.dev web site. I am new to SVG so maybe missing something basic. Any help appreciated.
[RendersOk.json](https://github.com/clientIO/joint/files/13746811/RendersOk.json)
![PathDataRenders](https://github.com/clientIO/joint/assets/32515524/b9f086ff-fe1b-49a3-bc88-f27dc85df610)
[RendersAsRectangle.json](https://github.com/clientIO/joint/files/13746812/RendersAsRectangle.json)

### Restrictions & Constraints

I have a large number of shapes to convert some with complicated paths so manually converting path data is something I'd like to avoid.

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by kumilingus** [comment]
The `standard.Path` uses `refD` attribute instead of `d`  in `v3.7` (not in upcomming `v4`). You need to reset the `refD` attribute.

```json
{
  "cells": [
    {
      "type": "standard.Path",
      "position": {
        "x": 300,
        "y": 150
      },
      "size": {
        "width": 60,
        "height": 60
      },
      "angle": 0,
      "id": "0e589c61-b458-4916-b3ea-536bf4764052",
      "attrs": {
        "body": {
          "refD": null, /* RESET */
          "d": "M20.44,0.206L0.263,20.382l19.411,19.412L39.85,19.616L20.44,0.206L20.44,0.206z"
        }
      }
    }
  ]
}

**Comment by gordon-brown** [comment]
Thanks so much for answering over the Christmas holidays I wasn’t expecting
 an answer for few days. Merry Christmas, and Happy New Year.

Gord
